src/xrpld/app/misc/LendingHelpers.h

```diff
@@ -25,22 +25,55 @@ roundPeriodicPayment(
     return roundToAsset(asset, periodicPayment, scale, Number::upward);
 }
 
-/// This structure is explained in the XLS-66 spec, section 3.2.4.4 (Failure
-/// Conditions)
+/* Represents the breakdown of amounts paid and changes applied during a loan
+ * payment.
+ *
+ * This structure is returned after processing a loan payment transaction and
+ * captures the actual amounts that were paid to each recipient (Vault and
+ * Borrower) and any changes to the loan's total value. It provides a complete
+ * accounting of where the payment funds went.
+ *
+ * This structure is explained in the XLS-66 spec, section 3.2.4.4 (Failure
```

Comment:
Is this right? 3.2.4.4 is `Transaction Pseudo-code`. But I don't see this structure description, at least not directly, in either section.

Reply:
Fixed.